apps/web/src/pages/quick-start/components/demo-flow/nodes/DigestNode.tsx

```diff
@@ -3,13 +3,19 @@ import { Handle, Position } from 'react-flow-renderer';
 
 import { NodeStep } from './NodeStep';
 import { DigestGradient } from '../../../../../design-system/icons/general/DigestGradient';
+import { colors } from '../../../../../design-system';
+import { useStyles } from '../../../../../design-system/template-button/TemplateButton.styles';
 
 export function DigestNode({ data, id }: { data: any; id: string }) {
+  const { theme } = useStyles();
+  const ActionItem = data.ActionItem as React.FC<any>;
+
   return (
     <NodeStep
       data={data}
       id={id}
       Icon={DigestGradient}
+      ActionItem={() => <ActionItem style={{ color: `${theme.colorScheme === 'dark' ? colors.B40 : colors.B80}` }} />}
```

Comment:
it's better to pass it as the ReactNode, not the component because otherwise, it will always create a new one... I've already changed it in the PR: #3013